File: cpp/examples/occupancy_map_registration.cpp
#include "registration_common.hpp"

int main() {
    auto set = examples::robot_description_fleet();
    set.datavizs.front().add_occupancy_grid(
        "/map",
        "map",
        {{"color_free", std::string("#222222")}, {"color_occupied", std::string("#F4F4F4")}, {"alpha", 0.85}});
    return examples::register_or_fail(set);
}

File: rust/examples/occupancy_map_registration.rs
mod common;

fn main() {
    let mut set = common::robot_description_fleet();
    set.datavizs[0].add_occupancy_grid(
        "/map",
        "map",
        Some(common::options(&[
            ("color_free", serde_json::json!("#222222")),
            ("color_occupied", serde_json::json!("#F4F4F4")),
            ("alpha", serde_json::json!(0.85)),
        ])),
    );
    common::register_or_fail(&mut set);
}
